internal/experiment/webconnectivityqa/control.go

```diff
@@ -89,8 +90,9 @@ func controlFailureWithSuccessfulHTTPSWebsite() *TestCase {
 		ExpectErr: false,
 		ExpectTestKeys: &testKeys{
 			ControlFailure: "unknown_failure: httpapi: all endpoints failed: [ connection_reset; connection_reset; connection_reset; connection_reset;]",
-			XStatus:        1, // StatusSuccessSecure
-			XNullNullFlags: 8, // analysisFlagNullNullSuccessfulHTTPS
+			DNSConsistency: "consistent",
```

Comment:
We conclude that the DNS is consistent because TLS works. This is not a regression, rather a way in which Web Connectivity LTE is outperforming v0.4.